projects/nexus-dashboard/day-2/Joyal-Poudel/src/styles/index.css

```
@@ -0,0 +1,362 @@
+  
+  --shadow-glow: 0 0 20px -5px var(--accent-glow);
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+html {
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.10.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)